tests/tacacs/test_ro_disk.py

```diff
@@ -54,15 +55,13 @@ def do_reboot(duthost, localhost, dutip="", rw_user="", rw_pass=""):
     rebooted = False
 
     for i in range(retries):
-        # Regular reboot command would not work, as it would try to
-        # collect show tech, which will fail in RO state.
         #
         try:
-            if dutip:
-                chk_ssh_remote_run(localhost, dutip, rw_user, rw_pass, "sudo /sbin/reboot")
-            else:
-                duthost.shell("/sbin/reboot")
-
+            # Reboot DUT using reboot function instead of using ssh_remote_run.
+            # ssh_remote_run gets blocked due to console messages from reboot on DUT
+            # Do not wait for ssh as next step checks if ssh is stopped to ensure DUT is
+            # is rebooting.
+            reboot(duthost, localhost, wait_for_ssh=False)
```

Comment:
Enabled AAA authentication failthrough so that the reboot works.
Are you referring to how the console messages from DUT are handled during this reboot? Because when we send it over SSH, reboot is a blocking call.

Reply:
I mistook it. 
Is this code change tested in couple of ansible test beds?